<?php

namespace ContainerJ4Zx9zr;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getContentfulCredentialsValidatorService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'App\Validator\Constraints\ContentfulCredentialsValidator' shared autowired service.
     *
     * @return \App\Validator\Constraints\ContentfulCredentialsValidator
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidator.php';
        include_once \dirname(__DIR__, 4).'/src/Validator/Constraints/ContentfulCredentialsValidator.php';

        return $container->privates['App\\Validator\\Constraints\\ContentfulCredentialsValidator'] = new \App\Validator\Constraints\ContentfulCredentialsValidator(($container->privates['App\\Service\\Contentful'] ?? $container->getContentfulService()), ($container->privates['App\\Service\\State'] ?? $container->getStateService()));
    }
}
